### ISSUE TITLE
Can't shut off Google Instant with Disconnect

### ISSUE THREAD

**Issue by garyjob** [disconnectme/disconnect#20]
1.Install Disconnect in Chrome, do no setting of Disconnet preferences or settings (haven't looked to see what you can or can't do)
2. Go to Google search and try to shut off instant. It forces you to google's preferences page where the Location field is required. No matter what valid location I enter, it raises an error dialog (see attached)
1. I tried setting Disconnect to allow Google then I entered a valid address to some podunk town in Montana and saved successfully and got no instant Google, but as soon as I block Google in Disconnect, the instant google comes back on.

Bottom line: seems like it's either live with google instant (which I don't like) or set Disconnect to allow Google (which I don't like).

Win 7 Home edition

Thanks!
## Notes by Gary

Issue was note encountered in the browsers running on Windows XP:
- Firefox
- Safari
- Chrome

Link to original Google Code Issue:
http://code.google.com/p/byoogle/issues/detail?id=50
